rosdep/python.yaml

```
@@ -6515,6 +6515,22 @@ python3-flask-cors:
   ubuntu:
     '*': [python3-flask-cors]
     bionic: null
+python3-flask-jwt-extended:
+  debian:
+    bullseye: [python3-python-flask-jwt-extended]
```

```suggestion
    '*': [python3-python-flask-jwt-extended]
```

Change this to a wildcard for future compatibility.